submissions/KostiantynOkhorzin/memory-game/script.js

```diff
@@ -0,0 +1,102 @@
+'use strict';
+
+const gameBody = document.querySelector('.game__body');
+let playerSteps = 0;
+const time = 1500;
+
+const getData = () => [
+    { imgSrc: "img/ananas.png", name: "ananas" },
+    { imgSrc: "img/apple.png", name: "apple" },
+    { imgSrc: "img/avocado.png", name: "avocado" },
+    { imgSrc: "img/banana.png", name: "banana" },
+    { imgSrc: "img/kiwi.png", name: "kiwi" },
+    { imgSrc: "img/lime.png", name: "lime" },
+    { imgSrc: "img/ananas.png", name: "ananas" },
+    { imgSrc: "img/apple.png", name: "apple" },
+    { imgSrc: "img/avocado.png", name: "avocado" },
+    { imgSrc: "img/banana.png", name: "banana" },
+    { imgSrc: "img/kiwi.png", name: "kiwi" },
+    { imgSrc: "img/lime.png", name: "lime" },
+];
+
+const styleList = {
```

Comment:
What is the purpose of this object? You don't use it in the `render()` method of a `GameCard` on line 35.